docs/proposals/2170-kubeflow-training-v2/README.md

```diff
@@ -279,6 +279,43 @@ type TrainJob struct {
 	Status TrainJobStatus `json:"status,omitempty"`
 }
 
+const (
```

Comment:
Can we define those const as `TrainJobConditionType` similar to Batch/Job: https://github.com/kubernetes/api/blob/master/batch/v1/types.go#L617 ?

Reply:
Oh, you are right.